### PR TITLE
Tim Busch Lab 02: staging before turn in, still working on Array of Arrays

### PR DESCRIPTION
**Pull request by GoldBeardSea**

### CODE REVIEW

File: src/ArrayOfArrays.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        int[][] weeklyMonthTemperatures = {
+                {66, 64, 58, 65, 71, 57, 60},
+                {57, 65, 65, 70, 72, 65, 51},
+                {55, 54, 60, 53, 59, 57, 61},
+                {65, 56, 55, 52, 55, 62, 57}
+        };
+        int maxVal = averageOfArrays(weeklyMonthTemperatures);
+        System.out.println(maxVal);
+    }
+    public static int averageOfArrays(int[][] weeklyMonthTemperatures) {
```
Comment (mollyfish) [inline src/ArrayOfArrays.java:12]: You can name this parameter something more generic

File: src/CalculatingAverages.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+    }
+    public static int averageArray(int[] array) {
+        int averageHolder = 0;
+        for (int i = 0; i < array.length; i++) {
+            averageHolder = averageHolder + array[i];
+        }
+        averageHolder = averageHolder / array.length;
+        return averageHolder;
+    }
```
Comment (mollyfish) [inline src/CalculatingAverages.java:15]: nice!

File: src/ContainsDuplicatesTest.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (mollyfish) [inline src/ContainsDuplicatesTest.java:1]: Consider writing another test for this that takes an array with no duplicates, just to be sure your function works for that, too :)
